CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.7.0]
```

Comment:
this doesn't match the conventional commit.

Reply:
since this package is still under major version 0. DO NOT bump the major version.

Reply:
I've checked other release PR, and it looks good.

Reply:
Ohh right, it should be 0.6.0 I think following the current scheme.

Reply:
Bumped git version config.